python/tank/authentication/login_dialog.py

```diff
@@ -666,7 +668,7 @@ def exec_(self):
         # to freeze, so only set the WindowStaysOnTopHint flag as this appears to not disable the
         # other flags.
         self.setWindowFlags(QtCore.Qt.WindowStaysOnTopHint)
-        return QtGui.QDialog.exec_(self)
+        return QtWidgets.QDialog.exec_(self)
```

Comment:
What about the `exec_` method? I though we were not supposed to use it anymore with recent syntax?

Reply:
Thanks for the detailed explanation @carlos-villavicencio-adsk!

However, switching from `exec_` to `open` is not straightforward because I am aware that, in some places (TK auth), we rely on `exec_` blocking call. So we would have to re-think the workflow to use `open`.